pallets/confidential/src/lib.rs

```diff
@@ -79,6 +88,36 @@ decl_module! {
 
         fn deposit_event() = default;
 
+        #[weight = 400_000]
```

Comment:
Weights are to be decided. Using 400,000 for everything for now.

Reply:
You can put weigh in a multiple of 1_000_000_000